internal/templates/template.go

```diff
@@ -23,16 +25,23 @@ func Init(w io.Writer, template, dir string) error {
 	}
 
 	templatePath, ok := templates[template]
+	cloneURL := "https://github.com/jetpack-io/devbox.git"
 	if !ok {
-		return usererr.New("unknown template %q", template)
+		u, err := url.Parse(template)
+		splits := strings.Split(u.Path, "/")
+		if err != nil || u.Scheme != "https" || len(splits) == 0 {
```

Comment:
splits needs to have more than zero no? only the org/repo name are 2 parts. Also, FYI this will return empty parts as well because Path starts with a "/"